modules/hooks.nix

```diff
@@ -67,10 +67,18 @@ in
 
   # PLEASE keep this sorted alphabetically.
   options.settings = {
-    rust.cargoManifestPath = mkOption {
-      type = types.nullOr types.str;
-      description = "Path to Cargo.toml";
-      default = null;
+    rust = {
+      check.cargoDeps = mkOption {
```

Comment:
@domenkozar, need to make a call here.
This is a setting that'll only be used to fix `nix flake check`. Other hooks might need to make their own modifications to the derivation used in flake check. 
Should we prefix them somehow or not?